doxygen/contributor_help_pages/getting_started.md

```diff
@@ -15,30 +14,34 @@ new code to keep the code from getting unwieldy (the old patterns will be
 updated eventually).
 
 The title contains "Current State" to emphasize that if any information here is
-out of date or any advice does not work, it should be reported as a bug (the
-[example-models](https://github.com/stan-dev/example-models)).
+out of date or any advice does not work, it should be reported as a bug (
+[issue tracker link](https://github.com/stan-dev/math/issues/new)).
 
-This document builds on the information in the [Stan Math Library](https://arxiv.org/abs/1509.07164) which should be skimmed over before .
+This document builds on the information in the [Stan Math Library](https://arxiv.org/abs/1509.07164). While some parts of the paper are out of date now it is very useful for understanding the patterns used in the math library.
 
 ## Preliminary Resources:
 
 Before contributing to Stan Math it's recommended you become familiar with C++. While this guide attempts to cover the edges and odd things we do in the math library, we recommend the resources below for getting started on autodiff in C++.
 
-Books:
+Books and Papers:
 - [Effective Modern C++](https://www.amazon.com/Effective-Modern-Specific-Ways-Improve/dp/1491903996)
-- [Bob's autodiff book](https://github.com/bob-carpenter/ad-handbook/blob/master/ad-handbook-draft.pdf)
```

Comment:
> I didn't mean drop the reference, I meant drop my name. So that'd be Bob's autodiff book -> The Autodiff Handbook (book titles are in italics). Of course, that's what JAX called their section, so it should technically be "An autodiff handbook". All the book titles should be italicized (that's *title* in markdown).

Yes I just placed it below in a different section 

> Also, I don't think Numerical Recipes is a good reference for autodiff or C++.

I agree I'll remove that